src/main/java/org/broadinstitute/hellbender/tools/exome/BayesianHetPulldownCalculator.java

```diff
@@ -170,11 +179,10 @@ public BayesianHetPulldownCalculator(final File refFile, final IntervalList snpI
      */
     public void useBalancedHetPrior() {
         hetPriorType = HetPriorType.BALANCED;
-        logger.info("Using the BALANCED Het prior.");
     }
 
     /**
-     * use the HETEROGENEOUS prior for Het sites.
+     * Use the HETEROGENEOUS prior for Het sites.
```

Comment:
Ok, 
perhaps you are going to kill me... but I think that it would be better if you abstract out the two types of het-priors into classes

```
abstract class HeterozygousPrior { 

  protected getLogLikelihoodFixedAlleleFraction(
                       final double f, final List<Double> alpha, final List<Double> beta) {
      ...
  }
  
  public abstract getLogLikelihoodWithHeterogeneousPrior(List<Double> alpha, List<Double> beta);
}

class BalancedHeterozygousPrior extends HeterozygousPrior {
  public abstract getLogLikelihood(List<Double> alpha, List<Double> beta) {
       return getLogLikelihoodFixedAlleleFraction(0.5, alpha, beta);
  }
}

class BalancedHeterozygousPrior extends HeterozygousPrior {
  @Override
  public  getLogLikelihood(List<Double> alpha, List<Double> beta) {
       return getLogLikelihoodFixedAlleleFraction(0.5, alpha, beta);
  }
}

class HomogeneousHeterozygousPrior extends HeterozygousPrior {

  public HomogenousHeterozygousPrior(final double minAbnormalFraction, final double maxAbnormalFraction,
                                          final int maxCopyNumber, final int quadratureOrder) {
     ...
  }

  @Override
  public  getLogLikelihood(final List<Double> alpha, final List<Double> beta) {
       return getLogLikelihoodFixedAlleleFraction(0.5, alpha, beta);
  }
}

```

Then ```useHeterozygousHetPrior``` and useBalanceHetPrior fuse ```usePrior(final HeterozygousPrior)```;